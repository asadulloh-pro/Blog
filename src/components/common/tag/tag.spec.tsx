import React from "react";
import { render } from "@testing-library/react";
import Tag from "./tag.component";

it("renders Tag component", () => {
  const { container } = render(<Tag>Смартфоны</Tag>);
  expect(container).toMatchSnapshot();
});
