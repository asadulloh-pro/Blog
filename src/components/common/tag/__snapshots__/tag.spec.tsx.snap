// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Tag component 1`] = `
<div>
  <span
    class="text-center text-[0.875rem] font-[500] leading-[1.25rem] px-[1rem] py-[0.62rem] rounded-[6.25rem] text-prime border border-solid border-prime cursor-pointer "
  >
    Смартфоны
  </span>
</div>
`;
